schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/spi/ActivityEndpoint.java

```diff
@@ -289,7 +320,8 @@ public Subscription getSubscription(User caller, @Named("userId") Key userId, @N
         mustBeAdminOrOrgMember(caller, createFromActivityId(activityId));
         checkFound(activityId);
         try {
-            return ActivityServiceFactory.getActivityService().getSubscriptions(activityId);
+            Activity activity = ActivityServiceFactory.getActivityService().getActivity(activityId);
+            return ActivityServiceFactory.getActivityService().getSubscriptions(activity);
```

Comment:
Again here, if possible, we should try to avoid loading the activity if it is not required....  No need to change now, as this will be done like that in the DAO.  Just conceptually